src/impl/channels.js

```diff
@@ -6,7 +6,28 @@ var dispatch = require("./dispatch");
 var MAX_DIRTY = 64;
 var MAX_QUEUE_SIZE = 1024;
 
-var CLOSED = null;
+var CLOSED = (function() {
+  var closedToken;
+  try {
+    closedToken = Symbol.for('@@csp/CLOSED');
+  } catch(e) {
+    closedToken = Object('@@csp/CLOSED');
+  }
+  return closedToken;
+})();
```

Comment:
Yeah, they'll be pretty much equivalent, and I don't mind changing it if people find that version more readable - to me it's six of one, half dozen of the other. Tough to remember a year later, but I likely wrote it with a variable after initially going without the IIFE, and adding it in to avoid a function-wide deopt.